### DISCUSSION TITLE
Scalar Open Sourced LLM to auto generate API Guides

### DISCUSSION THREAD

**Discussion by jillxuu** [scalar/scalar#2652] [Ideas]
On this page, https://scalar.com/#description/resources, it says:
> Once connecting your file our open sourced LLM will get to work automatically generating your API Guide and Reference.

Sounded like scalar has an open source LLM to automatically generate the API guides as well? i've tried on this page after uploading my OpenAPI json spec, and it worked. But i couldn't find this feature after i log in. 

Could you help pointing me to the right place or is this feature not publicly available at the moment?

**Comment by marclave** [comment]
hey @jillxuu ! we have this on the landing page for onboarding then also exists for creating new pages : )

so when you got to create a new page you can specify what you would like our writer to write : )

**Reply by jillxuu** [reply]
on the boarding page, it auto generated the whole api guide based on my api references json spec. but after i create a new page, is there a button i can create that auto generate the whole api guide? i didn't find it nor did i find anything AI related
